javascript/scss/spht.scss

```diff
@@ -158,4 +158,12 @@ em {
 
 .tooltip {
   max-width: 400px;
+}
+
+.habitat-canvas {
```

Comment:
Could you explain the effect and purpose of this style to me? Does it bet buried under the map/header?

Reply:
In order to stack the canvases within a div, they need to have "position: absolute" set, and this seemed like the simplest place to set it.

Reply:
Ah, ok. And then I'm guessing leaflet correctly sets left, top, and z-index when the tile gets added to the map.

Reply:
I think that left and top get remain set to 0 (since that's the position within the div), and that z-index gets overridden by the z-index of the div itself, which is set in leaflet's own css. We probably don't need to set it explicitly.